runtime/compiler/ilgen/J9ByteCodeIlGenerator.hpp

```diff
@@ -247,8 +247,8 @@ class TR_J9ByteCodeIlGenerator : public TR_IlGenerator, public TR_J9ByteCodeIter
    void         genMonitorEnter();
    void         genMonitorExit(bool);
    TR_OpaqueClassBlock *loadValueClass(int32_t classCpIndex);
-   void         genAconst_init(uint16_t classCpIndex);
-   void         genAconst_init(TR_OpaqueClassBlock *valueTypeClass);
+   void         genAconst_init(int32_t classCpIndex);
+   void         genAconst_init(TR_OpaqueClassBlock *valueTypeClass, int32_t cpIndex);
```

Comment:
It would probably make sense to clean up the signatures for the two `genWithField` methods and `genFlattenableWithField` to use `int32_t` instead of `uint16_t` for their `cpIndex` arguments.

Although I'm pretty sure they're always used with values in the range [0,65535] today, it might be worthwhile not to tempt fate and make the signatures consistent in that regard.

Reply:
Addressed in 223f29e